src/main/java/webserver/RequestHandler.java

```diff
@@ -18,26 +16,39 @@ public RequestHandler(Socket connectionSocket) {
         this.connection = connectionSocket;
     }
 
+    @Override
     public void run() {
-        log.debug("New Client Connect! Connected IP : {}, Port : {}", connection.getInetAddress(),
-                connection.getPort());
+        log.debug("New Client Connect! Connected IP : {}, Port : {}", connection.getInetAddress(), connection.getPort());
 
-        try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
+        try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(connection.getInputStream())); OutputStream out = connection.getOutputStream()) {
             // TODO 사용자 요청에 대한 처리는 이 곳에 구현하면 된다.
-            DataOutputStream dos = new DataOutputStream(out);
-            byte[] body = "Hello World".getBytes();
-            response200Header(dos, body.length);
-            responseBody(dos, body);
+            HttpRequest httpRequest = new HttpRequest();
+            httpRequest.write(bufferedReader);
+
+            HttpResponse httpResponse = UrlMapper.getResponse(httpRequest, bufferedReader);
```

Comment:
여깅에 매개 변수로 reader가 들어있는것도 조금 모호해 보입니다.
httpResonse는 Request 만으로 생성할 수 있게 수정해 보면 어떨까요?
```suggestion
            HttpResponse httpResponse = UrlMapper.getResponse(httpRequest);
```